src/DesktopTests/UIAutomation/CustomObjects/RegistrarTests.cs

```diff
@@ -42,46 +43,59 @@ public void RegisterCustomProperty_CorrectDataFlow()
             uiaRegistrarMock.VerifyAll();
         }
 
+        private CustomProperty CreateEmptyPropertyOfType(string type)
```

Comment:
Can the input parameter here be a CustomUIAPropertyType instead of a string? I'm guessing that the type was changed to get the lower case name for prop.ConfigType. Maybe `type.ToString().ToLowerInvariant()`, instead?

Reply:
No, because `Type` cannot be set. `Type` is set via `ConfigType` which is a string. I suppose I could `switch` on the `Type` enum and set `ConfigType` manually?